src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String name;
+        int[] salary = {0, 0, 0};
```

Comment:
we don't need these variables. Let's try to simplify this task: 
iterate names with for-each (and for each name iterate data - split row and check date, name) if row satifies us calculate salary and add it to total.
this will make code shorter, if you have problem - visit QnA we will help)